db/seeds.rb

```diff
@@ -5,3 +5,11 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+Team.create(name: 'Arizona Cardinals', short_name: 'Arizona', code: 'ARI')
+Team.create(name: 'Atlanta Falcons', short_name: 'Atlanta', code: 'ATL')
+Team.create(name: 'Baltimore Ravens', short_name: 'Baltimore', code: 'BAL')
+Team.create(name: 'Buffalo Bills', short_name: 'Buffalo', code: 'BUF')
+Team.create(name: 'Carolina Panthers', short_name: 'Carolina', code: 'CAR')
+Team.create(name: 'Chicago Bears', short_name: 'Chicago', code: 'CHI')
+Team.create(name: 'San Francisco 49ers', short_name: 'San Francisco', code: 'SF')
```

Comment:
Line is too long. [81/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Final newline missing.